pallets/stableswap/src/lib.rs

```diff
@@ -166,6 +174,9 @@ pub mod pallet {
 		#[pallet::constant]
 		type AmplificationRange: Get<RangeInclusive<NonZeroU16>>;
 
+		/// Oracle providing prices for asset pegs (if configured for pool)
+		type TargetPegOracle: RawOracle<Self::AssetId, Balance, BlockNumberFor<Self>>;
```

Comment:
ability to use only raw oracle seems like oddly specific, reason?

Reply:
we don't want the value that delayed, that's not how the mechanism is designed

added comment too.